tests/unittests/session_events_tests.py

```
@@ -0,0 +1,49 @@
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+from dataclasses import asdict
+import unittest
+
+from dacite import from_dict
+
+import fiftyone as fo
```

Ruff (0.8.2)

`fiftyone` imported but unused

Remove unused import: `fiftyone`

(F401)